pennylane/ops/qubit/qchem_ops.py

```diff
@@ -114,23 +122,23 @@ def label(self, decimals=None, base_label=None):
         return super().label(decimals=decimals, base_label=base_label or "G")
 
 
-class SingleExcitationMinus(Operation):
-    r"""SingleExcitationMinus(phi, wires)
-    Single excitation rotation with negative phase-shift outside the rotation subspace.
+class SingleExcitationPlus(Operation):
+    r"""SingleExcitationPlus(phi, wires)
```

Comment:
Did the order of the operations swap?

Reply:
Yes, exactly, in order to make it `..Plus` first, matching the `DoubleExcitation` order. Sorry for being pedantic :sweat_smile: